src/main/resources/db/changesets/20211007_snapshot_storage_account.yaml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
```

Comment:
Nice, I think this makes sense, especially since there is already a column in the snapshot for the app deployment id.